blocks/editable/patterns.js

```diff
@@ -47,6 +47,10 @@ export default function( editor ) {
 		canUndo = null;
 	} );
 
+	editor.on( 'pastepostprocess', () => {
+		setTimeout( space );
```

Comment:
That will be fine, but I agree it's better to separate. Still needs polishing :)